### DISCUSSION TITLE
Alexa Media Player "AlexapyLoginError("Errore di accesso rilevato; API non in contatto")"

### DISCUSSION THREAD

**Discussion by Tam9rat** [indomus/forum#3233] [HUB: Home Assistant]
Ciao a tutti, sono riuscito a configurare bene l'integrazione, tuttavia funziona per alcune volte, ma dopo un paio d'ore ha smesso di funzionare.

Ho usato un codice yaml per la conversione del testo in voce tramite Notifiche: Invia una notifica con il servizio alexa_media. Non conosco il problema, ho provato a esportare i miei cookie dal browser e incollarli nel percorso /config in base alla documentazione, e ho anche scritto le credenziali per Amazon nel mio configutation.yaml, ma il problema persiste.

Ho provato a eseguire il debug dal file di log "AlexapyLoginError("Errore di accesso rilevato; API non in contatto")

alexapy.errors.AlexapyLoginError: Errore di accesso rilevato; API non in contatto"

Ho superato questo problema riavviando l'assistente domestico, ma non è un buon approccio a lungo termine?

**Comment by Scialla** [comment]
il web è affollato di richieste di assistenza, l'integrazione NON funziona bene, lo ripetiamo oramai da parecchio tempo, strano non te ne sia accorto...

In ogni caso puoi tamponare disattivando il `polling` nelle opzioni di configurazione dell'integrazione, oppure puoi lasciarlo attivo ma aumenta il tempo di scansione da 60 a 600 secondi

**Comment by gsarcl366** [comment]
Buongiorno, a me addirittura seguendo la guida: Integrare Amazon Alexa come “Media Player” su Home Assistant, quando arrivo alla fase di integrazione di Alexa Media Player, attivato il 2FA e compilati i campi regolarmente in fase di integrazione, inserito l'OTP, nel momento in cui accedo alla nuova finesta di autenticazione sul mio account Amazon, non mi da la possibilità di autenticarmi e li mi blocco. E' capitato a qualcun altro? Qualche suggerimento? P.S tutti i passaggi precedenti sono corretti Giorgio Sarcletti

**Reply by KC-inDomus** [reply]
clicca su "crea nuovo account", anche se lo hai già
